public/css/print.css

```diff
@@ -1,47 +1,49 @@
+@font-face {
+	font-family: 'Lato';
+	src: url('Lato/Lato-Regular.ttf') format('truetype');
+}
+
 body {
-	font-family: Lato, 'Helvetica Neue', Helvetica, Arial, sans-serif;
-	font-size: 10px;
+	font-family: 'Lato', sans-serif;
+	font-size: 15px;
```

Comment:
Quite a few font-size: 15px in this commit - is it possible to get them into one or 2 places instead - especially if we aren't using sass vars or relative sizing.